app/services/hyrax/workflow/abstract_notification.rb

```diff
@@ -9,6 +31,10 @@ def self.send_notification(entity:, comment:, user:, recipients:)
 
       attr_reader :work_id, :title, :comment, :user, :recipients
 
+      # @param [Sipity::Entity] entity the Sipity::Entity that is a proxy for the relevant Hyrax work
+      # @param [?] comment
```

Comment:
This method is called here: https://github.com/samvera/hyrax/blob/520411c17903f3ab294fa7306d5877f1ccf82344/app/services/hyrax/workflow/notification_service.rb#L39 and in that class it specifies that comment is an object that responds to `#comment` https://github.com/samvera/hyrax/blob/520411c17903f3ab294fa7306d5877f1ccf82344/app/services/hyrax/workflow/notification_service.rb#L15